use_cases.md

```diff
@@ -1,5 +1,12 @@
 # Use Cases
+{"ID": "AC-1",
+"Name": "Viewing Stock Data",
+"Description": "Upon selecting a stock, the user is able to view data related to the stock",
+"Primary Actor": "User",
+"Pre-condition": "User has selected a stock",
+"Post condtion": "User is able to view data related to the selected stock",
+"Main success scenario": "1: user selects a stock
+                          2: systems displays data related to the stock"
```

Comment:
Since this is basically a list of flow states, I think it should be a list or a dict. To keep everything universal, I think this line should look like this:

"Main success scenario": {1: "user selects a stock",
2: "systems displays data related to the stock"}

Reply:
sure i can make the changes

Reply:
sweet!